.github/workflows/codeql-analysis.yml

```diff
@@ -40,7 +40,8 @@ jobs:
     #    uses a compiled language
 
     - run: |
-           sudo apt-get install --assume-yes  apache2-bin apache2 apache2-dev apache2-utils libmaxminddb-dev
+           sudo apt update
```

Comment:
Not that it matters too much or is likely to change, but if we used `apt-get` here we would avoid:

```
WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
```

Maybe we could change the ones in `test.yml` too.